cgi-bin/LJ/User/Message.pm

```diff
@@ -198,13 +205,26 @@ sub reset_email {
                 from    => $LJ::ADMIN_EMAIL,
                 subject => LJ::Lang::ml("email.emailreset.subject"),
                 body    => LJ::Lang::ml(
-                    "email.emailreset.body",
-                    {
-                        user     => $u->user,
-                        sitename => $LJ::SITENAME,
-                        siteroot => "$LJ::SITEROOT/",
-                        auth     => $auth
-                    }
+                    $changepass
+                    ? (
+                        "email.emailreset.body_withpasswd",
+                        {
+                            user     => $u->user,
+                            newpass  => $changepass,
+                            sitename => $LJ::SITENAME,
+                            siteroot => "$LJ::SITEROOT/",
```

Comment:
oh tfw

Reply:
yeah, the other way is probably better but oh well